### PR TITLE
TES-308: VM module

### PR DESCRIPTION
**Pull request by rdstn**
Adds a working basic VM module which can create a scaling group for GDB instances given a pre-existing virtual network.

### CODE REVIEW

File: modules/config/main.tf
Comment (mihailradkov) [inline modules/config/main.tf:1]: Let's leave only the modules that include terraform resources.

File: modules/vm/main.tf [hunk 1]
```diff
@@ -0,0 +1,187 @@
+data "azurerm_resource_group" "rg" {
```
Comment (mihailradkov) [inline modules/vm/main.tf:1]: These can be created with Terraform but let's do that in another MR.

File: modules/vm/main.tf [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    name                    = "graphdb_network_lb_ingress"
+    description             = "CIRDs allowed to access GraphDB."
+    priority                = 950
+    direction               = "Inbound"
+    access                  = "Allow"
+    protocol                = "Tcp"
+    source_port_range       = "*"
+    destination_port_range  = "7200"
+    source_address_prefixes = local.subnet_cidr_blocks
+    destination_address_prefix = "*"
```
Comment (mihailradkov) [inline modules/vm/main.tf:81]: Run `terraform fmt`

**Review by mihailradkov — APPROVED** [review]

File: modules/config/main.tf
Comment (rdstn) [reply modules/config/main.tf:1]: Wipe the unnecessary dirs, then?

Comment (mihailradkov) [reply modules/config/main.tf:1]: Yes

File: modules/vm/main.tf (hunk 2 above)
Comment (rdstn) [reply modules/vm/main.tf:81]: Done.

File: modules/vm/main.tf (hunk 1 above)
Comment (rdstn) [reply modules/vm/main.tf:1]: If we drop the RG, we pretty much have to do the network, subnets, etc. as well. Pretty much all data needs to go to resources. It can be a different MR.
